redis_store_test.go

```diff
@@ -233,6 +245,9 @@ func TestUserPresence(t *testing.T) {
 
 	store.MarkInactive("bazbar", "sock1")
 
+	// Give it time to propagate to Redis...
+	time.Sleep(1 * time.Second)
```

Comment:
We could build this in a way it waits for the command to finish on redis so we don't have to sleep, eh?